mmselfsup/models/utils/position_embedding.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+
+
+def get_2d_sincos_pos_embed(embed_dim, grid_size, cls_token=False):
```

Comment:
Why do we do it by numpy rather than torch? Can we unify it for simplicity or efficiency?

Reply:
Thank you for your valuable comments. I use torch to construct the pe, instead of numpy.